React/Base/RCTConvert.m

```diff
@@ -316,6 +316,13 @@ + (NSLocale *)NSLocale:(id)json
   @"underline line-through": @(RCTTextDecorationLineTypeUnderlineStrikethrough),
 }), RCTTextDecorationLineTypeNone, integerValue)
 
+RCT_ENUM_CONVERTER(RCTTextTransform, (@{
+  @"none": @(RCTTextTransformNone),
+  @"capitalize": @(RCTTextTransformCapitalize),
+  @"uppercase": @(RCTTextTransformUppercase),
+  @"lowercase": @(RCTTextTransformLowercase),
+}), RCTTextTransformNone, integerValue)
```

Comment:
The default value here should be `RCTTextTransformUndefined`, not `RCTTextTransformNone`.

@janicduplessis Right?

Reply:
Hmm not how this works, does [RCTTextAttributes applyTextAttributes] only receives changed attributes (and some undefined value for the ones that did not change)?

I think both options would work but I'm not sure if one makes more sense. Maybe check what we do for other enum text attributes.